spanner/stats.go

```diff
@@ -38,9 +38,7 @@ var (
 	tagCommonKeys    = []tag.Key{tagKeyClientID, tagKeyDatabase, tagKeyInstance, tagKeyLibVersion}
 
 	tagNumInUseSessions = tag.Tag{Key: tagKeyType, Value: "num_in_use_sessions"}
-	tagNumBeingPrepared = tag.Tag{Key: tagKeyType, Value: "num_sessions_being_prepared"}
-	tagNumReadSessions  = tag.Tag{Key: tagKeyType, Value: "num_read_sessions"}
-	tagNumWriteSessions = tag.Tag{Key: tagKeyType, Value: "num_write_prepared_sessions"}
+	tagNumSessions      = tag.Tag{Key: tagKeyType, Value: "num_sessions"}
```

Comment:
Would changing metrics be a breaking change? Should we give a deprecation notice?

Reply:
Done